README.md

```diff
@@ -1,2 +1,314 @@
+
+Let’s begin by creating and enabling the Dependency Dashboard. Since GitHub defaults to disable `issues` on forked repositories, we need to enable it on the forked RenovateTutorial repo:
+
+1) Navigate to the main page of the repo and go to `settings` -> `general`
+2) Check the `issues` checkbox under the Features section:
+
+<img width="1284" alt="issues settings" src="https://user-images.githubusercontent.com/102745725/178968523-fb002cf1-2510-4b4f-b840-f9776e660d92.png">
+
+- In order for the Dependency Dashboard to become available, we will need to re-run Renovate by triggering a webhook (for example, closing an update PR).
+  
+> **Note**
+> This is usually done in a click via the Dependency Dashboard.
+
+
+3) Go to the `Pull requests` section
+4) Select `Update dependency php to v8.1` and select `Close pull request`
+
+<img width="927" alt="close php" src="https://user-images.githubusercontent.com/102745725/178969009-7239db99-4abe-44d1-a86c-a0effdf6fb7a.png">
+
+5) This will trigger Renovate to run and the Dependency Dashboard will appear under the `Issues` section - navigate to it
+
+> **Note**
+> it may take a minute to appear.
+
+
+### The Dependency Dashboard includes:
+  - Overview of all updates that are still to-do:
+
+    - **Open** PRs
+    - **Rate Limited** - PRs blocked by rate limit setting and will be opened based on preferences.
+    - **Pending Approval** - PRs that require manual triggering based on configurations.
+    - **Awaiting Schedule** - PRs creation blocked by Renovate scheduling settings.
+    - **Pending Status Checks** - updates that await pending status checks in order to be created.
+  
+  - Visibility into **rejected/deferred updates**.
+  - List of all the **detected dependencies** and **package managers** in your repository.
+  
+  <img width="942" alt="Screen Shot 2022-07-14 at 14 05 40" src="https://user-images.githubusercontent.com/102745725/178968912-24ef22ec-fc98-4bf1-a293-9fb5dbf4c1b8.png">
+
+Users can manually trigger the creation of dependency updates directly from the dashboard. 
+
+You can also re-run the Renovate bot manually directly from the Dependency Dashboard by enabling the “Check this box to trigger a request Renovate to run again on this repository” option:
+
+ <img width="867" alt="rerun renovate" src="https://user-images.githubusercontent.com/102745725/178969114-c1b64333-b45a-4508-b638-1e25ad0adab5.png">
+ 
+
+Let’s dive into one of the dependency dashboard capabilities - **the Pending Approval feature**. 
+
+Say we want to take extra measures before updating major versions of a package (either to reduce noise or to handle it more carefully). Renovate offers an option to prevent automatic creation of major version update PRs and create such PRs only upon manual request from the Dependency Dashboard. 
```

Comment:
```suggestion
Say we want to take extra measures before updating major versions of a package (either to reduce noise or to handle it more carefully).
Renovate offers an option to prevent automatic creation of major version update PRs and create such PRs only upon manual request from the Dependency Dashboard. 
```